core/src/main/kotlin/com/justai/jaicf/channel/http/HttpBotRequest.kt

```diff
@@ -12,20 +12,26 @@ import java.nio.charset.Charset
  * @property parameters HTTP query parameters
  * @property requestMetadata optional metadata for request processing
  */
-data class HttpBotRequest(
```

Comment:
If you remove the data modifier, then add toString and equals/hashCode methods. For example, HttpBotChannelServlet:29 uses toString.

Reply:
I added toString for logging purposes, but equals/hashCode are not really needed. It should not be a data class by its semantics.